interapp-backend/package.json

```diff
@@ -43,9 +43,8 @@
   "dependencies": {
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
-    "express": "^4.19.2",
-    "express-async-errors": "^3.1.1",
-    "express-rate-limit": "^7.1.4",
+    "express": ">=5.0.0",
```

Comment:
actually i meant change the `>=` to `^`, sorry if that wasnt clear.

^version “Compatible with version”, will automatically update you to all future minor/patch versions that are backwards-compatible, without incrementing the major version. ^1.2.3 will use releases from 1.2.3 to < 2.0.0.

whereas >= 5.0.0 means 6.0.0 is allowed, which under semver means breaking changes

apply this to @types/express as well